tests/utils.py

```diff
@@ -475,7 +476,17 @@ class DummyWriter(DummyWriterMixin, AgentWriter):
     def __init__(self, *args, **kwargs):
         # original call
         if len(args) == 0 and "agent_url" not in kwargs:
-            kwargs["agent_url"] = "http://localhost:8126"
+            kwargs["agent_url"] = agent.get_trace_url()
+        if "api_version" not in kwargs:
```

Comment:
nit: `kwargs.get("api_version", "v0.5")` does the same thing but slightly prettier